documentation/design/drafts/input.md

```diff
@@ -0,0 +1,56 @@
+# Github Audit Plugin Input Config File
+
+Github audit plugin will be using config.yaml file for taking input from user
+
+## Sample Input Config
+```yaml
+providers:
+  github:       # type of provider , can be butbucket or gitlabs also or normal private self hosted git 
+  - account_name: maplelabs     #github account username
+    periodicity: 300        #preiodicity for pulling changes using github APIs
+    credentials:
+      username:     #either username or email is needed (mandatory)
+      email:
+      token: XXX        #github access token to be used as it is current github auth standard 
+    repo_list:      #list of repository names
+    - github_audit
```

Comment:
These repositories can be under different organization, users. Better to take full URL

Reply:
Better to organize in the below hierarchy.

Git Audit Jobs:
     Each Job configuration.
             Job Specific configuration: Name, interval, meta data etc which are in the context of this program.
             repo type: GitHub
             repo specific configuration: path, url, credentials for login specific to target repo.
             Job output configuration: 

System level configuration
targets

Example:

auditJobs:
        - Name:    apache-project-progress
          polling_interval: 300
          tags:
                    tag1: one
          repo_type: GitHub
          repo_name:  apache2
          repo_config:
                access_token:   XXXXXXKXKX
          output:
             - target_name: es1

targets:
     #configuration related to es1 used above.

Reply:
@msrivathsa , sure updating input yaml file as suggested

Reply:
@msrivathsa , added the changes in input.md file as suggested , please review